docs/user/tutorials/test.yaml

```diff
@@ -0,0 +1,29 @@
+  ```bash
```

Comment:
I don't see this file mentioned anywhere, think we can delete it

Reply:
I accidentally pushed this one 😄 removed